pages.zh/common/vim.md

```diff
@@ -23,16 +23,16 @@
 
 - 在文件中搜寻（按下 `n`/`N` 来在上一个 / 下一个结果中切换）：
 
-`<Esc> 更多信息： <Enter>`
+`<Esc> 更多信息：<Enter>`
 
 - 对整个文件使用正则表达式进行替换：
 
-`<Esc> 更多信息： <Enter>`
+`<Esc> 更多信息：<Enter>`
```

Comment:
I have no idea why this page is full of "More information" 🤔

Reply:
I used to review this page. I thought it means to ask users to see the information link. 😂

Reply:
Looks like I messed something up in #5240. I have no idea how, but with regex, anything is possible 😅

Reply:
I'll fix this page sometime.

Reply:
Thanks!